api/metrics/src/main/java/io/opentelemetry/api/metrics/GlobalMeterProvider.java

```diff
@@ -36,50 +24,7 @@ public static MeterProvider get() {
    * early as possible in your application initialization logic, often in a {@code static} block in
    * your main class.
    */
-  public static void set(MeterProvider meterProvider) {
-    globalMeterProvider.set(meterProvider);
-  }
-
-  /**
-   * Gets or creates a named meter instance from the globally registered {@link MeterProvider}.
-   *
-   * <p>This is a shortcut method for {@code getGlobalMeterProvider().get(instrumentationName)}
-   *
-   * @param instrumentationName The name of the instrumentation library, not the name of the
-   *     instrument*ed* library.
-   * @return a tracer instance.
-   */
-  public static Meter getMeter(String instrumentationName) {
-    return get().get(instrumentationName);
-  }
-
-  /**
-   * Gets or creates a named and versioned meter instance from the globally registered {@link
-   * MeterProvider}.
-   *
-   * <p>This is a shortcut method for {@code getGlobalMeterProvider().get(instrumentationName,
-   * instrumentationVersion)}
-   *
-   * @param instrumentationName The name of the instrumentation library, not the name of the
-   *     instrument*ed* library.
-   * @param instrumentationVersion The version of the instrumentation library.
-   * @return a tracer instance.
-   */
-  public static Meter getMeter(String instrumentationName, String instrumentationVersion) {
-    return get().get(instrumentationName, instrumentationVersion);
-  }
-
-  /**
-   * Creates a {@link MeterBuilder} for a named meter instance.
-   *
-   * <p>This is a shortcut method for {@code get().meterBuilder(instrumentationName)}
-   *
-   * @param instrumentationName The name of the instrumentation library, not the name of the
-   *     instrument*ed* library.
-   * @return a MeterBuilder instance.
-   * @since 1.4.0
-   */
-  public static MeterBuilder meterBuilder(String instrumentationName) {
-    return get().meterBuilder(instrumentationName);
+  public static void set(MeterProvider provider) {
+    globalMeterProvider = (provider == null) ? NoopMeterProvider.getInstance() : provider;
```

Comment:
Equating `null` = `NoopMeterProvider.getInstance()` is strange IMO. 

I think this should return or throw if null is passed. Throwing is inconsistent with current behavior, so I say return.